Silicon/AlderlakePkg/FspBin/FspBin.inf

```diff
@@ -23,8 +23,6 @@
   AlderLakeFspBinPkg/IoT/AlderLakeS/Include/FspmUpd.h         : Silicon/AlderlakePkg/Adls/Include/FspmUpd.h
   AlderLakeFspBinPkg/IoT/AlderLakeS/Include/FspsUpd.h         : Silicon/AlderlakePkg/Adls/Include/FspsUpd.h
   AlderLakeFspBinPkg/IoT/AlderLakeS/Include/FspInfoHob.h      : Silicon/AlderlakePkg/Adls/Include/FspInfoHob.h
-  AlderLakeFspBinPkg/IoT/AlderLakeS/VBT/Vbt.bin               : Platform/AlderlakeBoardPkg/VbtBin/Vbt.dat
-  AlderLakeFspBinPkg/IoT/AlderLakeS/VBT/Vbt.json              : Platform/AlderlakeBoardPkg/VbtBin/Vbt.json
```

Comment:
It seems customer will have no way to get JSON files. Or do we expect to provide JSON file along with VBT binary in the VbtBin folder for future update? Otherwise, it is not able to config VBT via DisCon tool.

Reply:
agree, i'll keep line 27 in this case